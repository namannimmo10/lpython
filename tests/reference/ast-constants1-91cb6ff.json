{
    "basename": "ast-constants1-91cb6ff",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/constants1.py",
    "infile_hash": "ca0a78d7946779e18980611ff4b95bf6123e2fba6f641463cbe70583",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-constants1-91cb6ff.stdout",
    "stdout_hash": "dc5c00c38bf157dc376ad01c5062463648f5a81b6043c63f56f93087",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}