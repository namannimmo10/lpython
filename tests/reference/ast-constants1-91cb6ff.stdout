(Module [(FunctionDef test_boz ([] [] [] [] [] [] []) [(AnnAssign (Name b Store) (Name str Load) () 1) (Assign [(Name b Store)] (Call (Name bin Load) [(ConstantInt 5 ())] []) ()) (Assign [(Name b Store)] (Call (Name bin Load) [(ConstantInt 64 ())] []) ()) (Assign [(Name b Store)] (Call (Name oct Load) [(ConstantInt 8 ())] []) ()) (Assign [(Name b Store)] (Call (Name oct Load) [(ConstantInt 56 ())] []) ()) (Assign [(Name b Store)] (Call (Name hex Load) [(ConstantInt 42 ())] []) ()) (Assign [(Name b Store)] (Call (Name hex Load) [(ConstantInt 12648430 ())] []) ())] [] () ()) (FunctionDef test_ord_chr ([] [] [] [] [] [] []) [(AnnAssign (Name s Store) (Name str Load) () 1) (AnnAssign (Name a Store) (Name i32 Load) () 1) (Assign [(Name a Store)] (Call (Name ord Load) [(ConstantStr "5" ())] []) ()) (Assign [(Name s Store)] (Call (Name chr Load) [(ConstantInt 43 ())] []) ())] [] () ()) (FunctionDef test_abs ([] [] [] [] [] [] []) [(AnnAssign (Name a Store) (Name i32 Load) () 1) (Assign [(Name a Store)] (Call (Name abs Load) [(ConstantInt 5 ())] []) ()) (Assign [(Name a Store)] (Call (Name abs Load) [(ConstantBool .false. ())] []) ()) (AnnAssign (Name b Store) (Name f32 Load) () 1) (Assign [(Name b Store)] (Call (Name abs Load) [(ConstantFloat 5.500000 ())] []) ())] [] () ())] [])
